op-e2e/actions/proofs/pectra_blob_schedule_test.go

```diff
@@ -0,0 +1,66 @@
+package proofs_test
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	legacybindings "github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+func TestPectraBlobSchedule(gt *testing.T) {
+	t := actionsHelpers.NewDefaultTesting(gt)
+	testCfg := &helpers.TestCfg[any]{
+		Hardfork: helpers.Holocene,
+	}
+	setBlobSchedule := func(dc *genesis.DeployConfig) {
+		dc.L1PragueTimeOffset = ptr(hexutil.Uint64(0))
+		// fix blob schedule after 2 L1 blocks
+		dc.L2GenesisPectraBlobScheduleTimeOffset = ptr(hexutil.Uint64(24))
+		dc.L1GenesisBlockExcessBlobGas = ptr(hexutil.Uint64(1e8))
+	}
```

Comment:
Be nice to run a sub test with no optional `PectraBlobSechduleTime` set. To cover what we are expecting with mainnet.

Reply:
Good idea, added as a test variant